module/documents/actor/actor.mjs

```diff
@@ -689,7 +689,7 @@ export default class Actor5e extends Actor {
     // Configure prototype token settings
     const s = CONFIG.DND5E.tokenSizes[this.system.traits.size || "med"];
     const prototypeToken = {width: s, height: s};
-    if ( this.type === "character" ) Object.assign(prototypeToken, {vision: true, actorLink: true, disposition: 1});
+    if ( this.type === "character" ) Object.assign(prototypeToken, {sight: { enabled: true }, actorLink: true, disposition: 1});
```

Comment:
Standard style limits line lengths to 120 characters, so this will need to go into a new line:

```suggestion
    if ( this.type === "character" ) Object.assign(prototypeToken, {
      sight: { enabled: true }, actorLink: true, disposition: 1
    });
```

Other than that it looks good.